cayenne-server/src/main/java/org/apache/cayenne/reflect/BeanAccessor.java

```diff
@@ -152,4 +163,19 @@ public void setValue(Object object, Object newValue) throws PropertyException {
 			throw new PropertyException("Error writing property: " + propertyName, this, object, th);
 		}
 	}
+	
```

Comment:
Won't these methods generate an NPE if the property name is a single character?

Reply:
No, this is basically the same implementation as before (just moved from the constructor to a method) so it should be functionally equivalent.